openvpn/src/com/untangle/app/openvpn/OpenVpnManager.java

```diff
@@ -795,10 +796,19 @@ private void writeRemoteServerFiles(List<OpenVpnRemoteServer> remoteServers)
 
                 // if user+pass auth is enabled create the auth file
                 if (server.getAuthUserPass()) {
+                    if(server.getAuthPassword()!= null){
```

Comment:
I don't understand the logic here.  We should be doing a conversion of the settings file on app start to convert the unencrypted password to encrypted, then use the encrypted only here.